net-mgmt/collectd/src/opnsense/mvc/app/models/OPNsense/Collectd/General.xml

```diff
@@ -113,5 +113,29 @@
             <default>1</default>
             <Required>N</Required>
         </p_users_enable>
+        <p_openvpn_enable type="BooleanField">
+            <default>0</default>
+            <Required>N</Required>
+        </p_openvpn_enable> 
+        <p_openvpn_statusfile type="TextField">
```

Comment:
I would make this field Required Y and set a default, it saves time

Reply:
isn't this the control socket ? if yes, we don't need a setting... we shouldn't support running / monitoring of stray openvpn instances here (openvpn not controlled by opnsense)

Reply:
No, control socket is in /var/etc/openvpn ... this is only a status

```
--status file [n]
Write operational status to file every n seconds.
Status can also be written to the syslog by sending a SIGUSR2 signal.

--status-version [n]
Choose the status file format version number. Currently n can be 1, 2, or 3 and defaults to 1.
```

Reply:
But it's perhaps not a bad idea to enable this by default with every server instance in the specific directory, then there's no need for this advanced hack :)

Reply:
so the status file is added with the advanced options? ouch.. it really only makes sense to fill this here if we have proper core support for these two options